pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -263,49 +261,25 @@ spec:
         - bash
         - -c
         - |-
-          # Capture all inbound and outbound traffic
```

Comment:
Why did you remove this ? Did we drop support for cluster.env and those settings ? Upgrade ?

Reply:
Rather than generating it "by hand" in the deployments command, we're generating it with istioctl and mounting it as a file. This matches what we have documented for users (besides the mounting details).